windows-terminal/config/configurePwsh7.ps1

```diff
@@ -36,7 +36,8 @@ else
     Write-Host "It appears that you already have $PROFILE file..." -Foreground Green
     Write-Host
     Write-Host "Add these as first lines of your powershell profile (that will open using Notepad):"
-    Write-Host "Invoke-Expression (oh-my-posh --init --shell pwsh --config ""$(scoop prefix oh-my-posh3)/themes/rebel-paradox.omp.json"")" -Foreground Yellow
+    Write-Host "`$env:POSH_GIT_ENABLED = `$true" -Foreground Yellow
```

Comment:
Be aware while this offers autocomplete, it also runs git status twice if you're not also using the [posh-git](https://ohmyposh.dev/docs/segments/poshgit) segment in favour of the git segment. In Windows, this is really slowing down the experience.

Reply:
Thank you for the advise... It's really appreciated ❤... 
I'll have to do some adaptations then since I use the "git segment" and not the posh-git in the custom theme... 😉